### PR TITLE
Removed larger input files 

### PR DESCRIPTION
**Pull request by SURUTHI1605**
Removed larger input files in fft2d design

### CODE REVIEW

File: AI_Engine_Development/Design_Tutorials/03-beamforming/sample_env_setup.sh~HEAD [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
Comment (imrickysu) [inline AI_Engine_Development/Design_Tutorials/03-beamforming/sample_env_setup.sh~HEAD]: This file doesn't seem to be needed.

File: AI_Engine_Development/Design_Tutorials/03-beamforming/sample_env_setup.sh~HEAD_0 [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
Comment (imrickysu) [inline AI_Engine_Development/Design_Tutorials/03-beamforming/sample_env_setup.sh~HEAD_0]: This file doesn't seem to be needed.

**Comment by imrickysu** [comment]
Hi @SURUTHI1605 , the target of this PR isn't clear to me.

1. The changed 48 files involves multiple tutorial designs. One PR should only contain one design update, or a batch of simple update for multiple tutorials, e.g., copyright update for all tutorials.
2. The changed files contain a lot of changes for 2022.2, but this PR is targeting to 2022.1 branch.
E.g. `AI_Engine_Development/Design_Tutorials/06-fft2d_AIEvsHLS/README.md`: "The `XLNX_VERSAL` path to the `xilinx-versal-common-v2022.2` directory is used in the step below."
Please confirm that you're committing the contents for the correct target branch. Thanks.

**Comment by SURUTHI1605** [comment]
Hi @imrickysu , I have changed files in FFT and FIR designs alone. I think because of git rebase some files of other designs got modified.

**Comment by imrickysu** [comment]
Hi @SURUTHI1605 , please take a look at github calculated changed files: https://github.com/Xilinx/Vitis-Tutorials/pull/316/files

For any files you don't intend to change, please follow the following instructions to remove them from this PR file change list.
https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

**Comment by SURUTHI1605** [comment]
Hi @imrickysu , 
Many changes have been done in FFT tutorials.So,all the 46 files has to be pushed in.

**Pull request closed without merging**